annotation-utils/src/main/java/edu/isi/vista/annotationutils/RestoreJson.kt

```diff
@@ -1,109 +1,148 @@
-package edu.isi.vista.annotationutils
```

Comment:
What happened to make this giant diff for this file? It makes it hard to review for changes.

Reply:
I'm not sure why this happened. Is there any other way I can highlight the changes?